### PR TITLE
Add xmlrpc-c

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-c-cpp - this is ready for review. Thanks!

/cc @wolfv - this is needed for some ROS packages in RoboStack

File: recipes/xmlrpc-c/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+./configure \
+    --prefix="${PREFIX}"  \
+    --enable-libxml2-backend
+
+make
```
Comment (xhochy) [inline recipes/xmlrpc-c/build.sh:7]:
```suggestion
make -j${CPUC_COUNT}
```

File: recipes/xmlrpc-c/build.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+make install
+
+pushd tools
+make
```
Comment (xhochy) [inline recipes/xmlrpc-c/build.sh:11]:
```suggestion
make -j${CPUC_COUNT}
```

**Review by xhochy — COMMENTED** [review]
LGTM but please use parallelisation where possible.

**Review by xhochy — APPROVED** [review]

**Comment by xhochy** [comment]
@Tobias-Fischer Sadly only realized after merging: this is missing a `run_exports`. Can you please add that on the feedstock?

**Comment by wolfv** [comment]
ah, sorry, I should have seen it as well.

**Comment by wolfv** [comment]
we should have a lint for that :/

**Pull request merged**